doesn't compile

$ make
cc -Wall      -o katz -lrt katz.c
katz.c: In function ‘connected_udp_socket’:
katz.c:939:9: warning: format not a string literal and no format arguments [-Wformat-security]
katz.c: In function ‘katz_connect’:
katz.c:1111:9: warning: format not a string literal and no format arguments [-Wformat-security]
katz.c:1113:9: warning: format not a string literal and no format arguments [-Wformat-security]
/tmp/ccXig6vP.o: In function `katzconn_getq':
katz.c:(.text+0xe67): undefined reference to `clock_gettime'
katz.c:(.text+0xefa): undefined reference to `clock_gettime'
/tmp/ccXig6vP.o: In function `katzconn_keepalive':
katz.c:(.text+0x105f): undefined reference to `clock_gettime'
/tmp/ccXig6vP.o: In function `smallest_oq_timeout':
katz.c:(.text+0x114c): undefined reference to `clock_gettime'
/tmp/ccXig6vP.o: In function `katz_process_out':
katz.c:(.text+0x160f): undefined reference to `clock_gettime'
/tmp/ccXig6vP.o:katz.c:(.text+0x16d7): more undefined references to `clock_gettime' follow
collect2: ld returned 1 exit status
make: *** [katz] Error 1

ubuntu linux 12.04

--- a/Makefile
+++ b/Makefile
@@ -8,7 +8,7 @@ CFLAGS=-Wall
 CFLAGS+=$(DEBUG) ${PROFILE} $(DEFINES) $(LIBS) 
 
 katz: katz.c katz.h
-	$(CC) $(CFLAGS) -o katz katz.c
+	$(CC) -o katz katz.c $(CFLAGS)
 
 debug: katz
 

